### PR TITLE
Feature/feature main menu

### PR DESCRIPTION
**Pull request by xavier-campos-diaz**

### CODE REVIEW

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/activities/views/PetComponentView.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5231571&bid=17151074)

File: app/src/main/java/org/pesmypetcare/mypetcare/activities/MyPetsComunication.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package org.pesmypetcare.mypetcare.activities;
+
+import org.pesmypetcare.mypetcare.features.users.User;
+
+public interface MyPetsComunication {
+
+    User getUser();
```
Comment (AlbertPG) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/MyPetsComunication.java:11]: You need to document this function

File: app/src/main/java/org/pesmypetcare/mypetcare/activities/views/MainMenuView.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        Space space;
+        Pet currentPet = userPets.remove(0);
+        ConstraintLayout component = new PetComponentView(currentActivity, null).initializePetComponent(currentPet);
+        this.addView(component);
+        this.petComponents.add(component);
+        space = initializeSpacer();
+        this.addView(space);
+    }
+
+    private Space initializeSpacer() {
```
Comment (AlbertPG) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/views/MainMenuView.java:66]: You need to document this function as well

**Review by AlbertPG — COMMENTED** [review]
You should change this

**Review by AlbertPG — APPROVED** [review]
Everything is correct

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/activities/views/PetComponentView.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5231571&bid=17151074)

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/activities/views/PetComponentView.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5231571&bid=17151074)

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/activities/views/PetComponentView.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5231571&bid=17151074)

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/activities/views/PetComponentView.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5231571&bid=17151074)

**Pull request merged**